src/Fusion.Summary.Api/Deployment/k8s/pr-deployment-env.yml

```diff
@@ -24,6 +24,8 @@ metadata:
   labels: 
     environment: pr
     prNumber: '{{prNumber}}'
+  annotations:
+    k8s-ttl-controller.twin.sh/ttl: '3d'    
```

Comment:
@HansDahle Does new commits extend the time?

Reply:
3 days seems a little low in order to cover the need. Work should be "quick and easy", but I think we need to accommodate edge cases. 3 days * PI = 9? Or do you want to stick to 3 and see how it goes? But it feels a bit short, with weekends you might need to spin up again Monday if you had multiple tasks going before the weekend and did not complete testing for instance.

Reply:
Typically you test the service with postman.. if you discover that it does not respond, it's just to go in and re-run the check or push a new commit. 

Consider it from a different point of view, if all teams builds api for each pr, and it lives 9 days, where you only test here and there, maybe the same day you develop, how much stuff would just lie in the cluster, hogging capacity? Every api will "resevere" capacity, which means that the cluster would have to scale up, just for primarily idle stuff... E.g. snyk/dependabot prs.. 

We will start here, if it becomes an issue, we adjust